REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.15
+Compat 0.9.4
```

Comment:
what are you going for with 0.9.4? that only introduced the xor unicode operator: https://github.com/JuliaLang/Compat.jl/compare/v0.9.3...v0.9.4